components/usage/pkg/apiv1/usage.go

```diff
@@ -24,10 +24,17 @@ import (
 
 var _ v1.UsageServiceServer = (*UsageService)(nil)
 
+type UsageConfig struct {
+	DefaultSpendingLimitForUsers int32 `json:"defaultSpendingLimitForUsers,omitempty"`
+
+	DefaultSpendingLimitForTeams int32 `json:"defaultSpendingLimitForTeams,omitempty"`
+}
+
 type UsageService struct {
 	conn    *gorm.DB
 	nowFunc func() time.Time
 	pricer  *WorkspacePricer
+	cfg     UsageConfig
```

Comment:
I think passing in the `UsageConfig` is exposing too much, I think Maybe just passing in something like 
```
type SpendingLimitDefaults strut {
   Teams, Individuals  int
}
```
would be sufficient here.

Reply:
`UsageConfig` is a new type I introduced for this.

Reply:
Ah, my bad. I took as the UsageComponentConfig. Ignore then.